src/aig-common/modules/standard/components/ipp-procedure/ipp-procedure-form/ipp-procedure-form.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
this component not follow the standard indication of NewUpdateForm.
Fix it

Reply:
This component also was modified as *-new-update-form component.